components/load_from_hf_hub/src/main.py

```diff
@@ -57,6 +58,9 @@ def load(self) -> dd.DataFrame:
                     (subset_field_name, subset_field_name)
                 columns.append(column_name)
 
+        if self.index_column is not None:
```

Comment:
This is needed to keep the index column that will be dropped later on (if it exists). We weren't picking up on it since it was not in the component spec or remapping dict